Core.lua

```diff
@@ -193,6 +193,10 @@ function WarpDeplete:Show()
   self.frames.root:Show()
   self:UpdateLayout()
 
+  if IsAddOnLoaded('SylingTracker') then
```

Comment:
I would prefer if we did this the same way as with `KT`, see above in the same file:

```lua
local KT = LibStub("AceAddon-3.0"):GetAddon("!KalielsTracker", true)
```

Then, you could add a comment there as well, explaining what needs to be done differently in case the AddOn is found.